Developer Samples/WindowsContainers/inrule-catalog-manager/README.md

```diff
@@ -0,0 +1,3 @@
+# Catalog Manager Container Sample
+
+WIP
```

Comment:
This guy needs the same great content as the other images.

Reply:
added basic README info in my latest push -- anything major missing?